### PR TITLE
Update LexDAO space (token) 

### PR DESCRIPTION
**Pull request by Ro5s**
update dao token reference to 0x4053981a40836141Cb79F05f68E9da385ff1814E

### CODE REVIEW

**Comment by Ro5s** [comment]
also, if possible, how might I include update logo.png - https://imgur.com/gallery/fRrvQGJ ?

**Comment by gokaiorg** [comment]
> also, if possible, how might I include updated logo.png - https://imgur.com/gallery/fRrvQGJ ?

Replace them on your local fork folder and commit the changes. You can use doc to see how update your local fork https://docs.snapshot.page/guides/create-a-space#1-fork-snapshot-spaces-repository-here

File: spaces/lexdao/index.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "token": "0x057e820D740D5AAaFfa3c6De08C5c98d990dB00d",
+  "token": "0x4053981a40836141Cb79F05f68E9da385ff1814E",
```
Comment (gokaiorg) [inline spaces/lexdao/index.json:2]: If you change the base token address your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

**Review by gokaiorg — APPROVED** [review]

File: spaces/lexdao/index.json (hunk 1 above)
Comment (gokaiorg) [reply spaces/lexdao/index.json:2]: We are doing some changes for spaces settings, you can change your strategy token if you want but the root token value must stay like that.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
We've made a major global update, please pull before commit again your changes.
You can use the documentation to help you.
https://docs.snapshot.page/guides/create-a-space#1-fork-snapshot-spaces-repository-here

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**